engine/travis_script.sh

```diff
@@ -6,4 +6,4 @@ which gcc
 which gnatchop
 which gprbuild
 
-make tests html
+make SPHINXOPTS="-W" tests html
```

Comment:
The build test failed with this option on a warning in the Ada for CPP Java book. That book might require a bit of syntax cleanup before this option will allow the automated tests to complete.